go.mod

```diff
@@ -3,8 +3,8 @@ module antrea.io/antrea
 go 1.19
 
 require (
-	antrea.io/libOpenflow v0.9.2
-	antrea.io/ofnet v0.6.10
+	antrea.io/libOpenflow v0.10.1
```

Comment:
Probably add a description on the PR and commit message about version upgrade?

Reply:
Done.